script/seed.js

```diff
@@ -12,6 +50,12 @@ async function seed() {
     User.create({email: 'murphy@email.com', password: '123'})
   ])
 
+  await Promise.all(
+    candies.map(candy => {
+      return Candies.create(candy)
+    })
+  )
+
   console.log(`seeded ${users.length} users`)
   console.log(`seeded successfully`)
```

Comment:
minor. it would be useful to add a console log with the number of candies that are added.